plugins/jobs/general/jobs_general_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"golang.org/x/exp/slog"
 )
 
+const addr = "127.0.0.1:6001"
```

Comment:
Do not use address as const here, because they should be rotated per test (in my TODO list). Let's undo this change.
When address is deallocated at the end of the test, it doesn't disappear from the OS at the same time. Instead, there is a period of time for the cleanup routine. To avoid failures in the following tests, it's better to use different TCP ports instead of making them constant.

Reply:
Ok, reverted